app/javascript/components/issue_monogram_form.vue

```diff
@@ -0,0 +1,128 @@
+<template>
+  <div>
+    <input
+      name="numismatics_issue[numismatic_monogram_ids][]"
+      type="hidden"
+      :value="id"
+    >
+
+    <form
+      id="nested_new_numismatics_monogram"
+      :action="action"
+      :method="method"
+      accept-charset="UTF-8"
+      enctype="application/json"
+      class="simple_form new_numismatics_monogram"
+      @created="created"
+      @submit.prevent="submit($event)"
+    >
+      <input
+        name="utf8"
+        type="hidden"
+        value="✓"
+      />
+
+      <div
+        id="metadata"
+        class="form-panel-content"
+      >
+        <div class="panel-body">
+          <div class="form-group string required numismatics_monogram_title">
+            <label
+              for="numismatics_monogram_title"
+              class="control-label string required"
+            >
+              <span class="label label-info required-tag">required</span> Title
+            </label>
+
+            <input
+              name="numismatics_issue[numismatic_monogram_ids][]"
+              type="hidden"
+              :value="id"
+            />
+
+            <input
+              id="numismatics_monogram_title"
+              v-model="title"
+              aria-required="true"
+              type="text"
+              name="numismatics_monogram[title]"
+              class="form-control string required form-control"
+            />
+          </div>
+        </div>
+
+        <div class="panel-save-controls">
+          <input
+            type="submit"
+            name="commit"
+            value="Save"
+            class="btn btn-primary save"
+            :disabled="disabled"
+          />
+        </div>
+      </div>
+    </form>
+  </div>
+</template>
+<script>
+import axios from 'axios'
```

Comment:
I'm using `axios` rather than `fetch` here with the hope that these can eventually be GraphQL queries in the future.